src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Figure {
```

Comment:
As we can implement all required interfaces
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

and getColor becomes just a regular getter